### PR TITLE
feat(web): Convo View

### PR DESCRIPTION
**Pull request by McPizza0**
Adds the convo view panel according to new designs

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#485** <a href="https://app.graphite.dev/github/pr/un/inbox/485?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by BlankParticle** [comment]
image
There is extra whitespace in the convos list as the grid is divided in 3 cols and convos list occupies 1/3 of that, but the convos list has a max width which causes the issue

**Comment by BlankParticle** [comment]
image
while having more messages the send sections goes out of the screen

File: apps/web/src/components/avatar.tsx [hunk 1]
```diff
@@ -127,11 +115,82 @@ export function Avatar(props: AvatarProps) {
...
+
+  return (
+    <div
+      className={cn(
+        'flex flex-row items-center gap-1',
+        avatarIconVariants({ size: iconProps.size })
+      )}>
+      {iconProps.withDot && <Dot />}
+      {icon()}
+      <span className="">{text()}</span>
```
Comment (BlankParticle) [inline apps/web/src/components/avatar.tsx:193]: Same as the last comment, call them like `<Icon/>` and `<Text/>` or you can also calculate the value and store them in `icon` and use it like `{icon}` not calling as function

File: apps/web/src/components/shared/attachments.tsx [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+  return (
+    <div
+      className="group relative w-fit p-1"
+      key={attachment.publicId}>
+      <div className="border-base-6 group-hover:bg-base-3 flex flex-row gap-2 rounded-md border px-2 py-1.5">
+        <div
+          className={cn(
+            iconClasses({ color: fileType as IconClassProps['color'] })
+          )}>
+          {icon()}
```
Comment (BlankParticle) [inline apps/web/src/components/shared/attachments.tsx:364]: this too

**Review by BlankParticle — CHANGES_REQUESTED** [review]

**Comment by McPizza0** [comment]
@BlankParticle updated

**Review by BlankParticle — APPROVED** [review]
Message UI overflow issues aside, LGTM

**Comment by McPizza0** [comment]
### Merge activity

* **Jun 19, 7:44 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/485).

**Pull request merged**